docs/pages/project/contributing/contributing-error.md

```diff
@@ -142,7 +142,7 @@ var (
 func main() {
 	log, err := logger.New("test", logger.Options{
```

Comment:
@Aviral0702 You can take a look to this set the `logLevel` from [here ](https://github.com/meshery/meshery/blob/master/server/machines/grafana/register.go#L25)